docs/index.adoc

```diff
@@ -592,6 +592,115 @@ public class SecuredServiceEndpoint {
 
 For more details see this guide: https://openliberty.io/guides/microprofile-jwt.html[Securing microservices with JWTs]
 
+== Messaging
+
+Applications can pass messages among themselves with the Java Message Service (JMS) APIs. Liberty can act as a server of JMS destinations (queues and topics,) or it can connect to another server to access those destinations--either another Liberty server or IBM MQ.
+
+=== JMS Server
+Liberty can act as a JMS server whether or not it is serving any applications. 
+
+.server.xml
+[source,xml]
+----
+<featureManager>
+    <feature>wasJmsServer-1.0</feature>
+</featureManager>
+
+<messagingEngine>
+    <queue id="libertyQueue" />
+    <topicSpace id="libertyTopic" />
+</messagingEngine>
+----
+
+If you want applications on other servers to connect to your JMS server, create a `wasJmsEndpoint`. Without this, only applications served by the same Liberty server can acccess the JMS destinations.
+
+.server.xml
+[source,xml]
+----
+<wasJmsEndpoint id="InboundJmsCommsEndpoint" 
+                host="*" 
+                wasJmsPort="7276" 
+                wasJmsSSLPort="7286" />
+----
+
+=== JMS Client
```

Comment:
This section has some good config snippets in it, but lets finish connecting the dots by (concisely) showing how an application might utilize this configuration. No need to have a 1-1 Java example for each config element, but maybe something that demonstrates how to send/receive a simple JMS message?